src/containers/AssetForm.test.js

```diff
@@ -38,57 +40,18 @@ const NEW_ASSET_FIXTURE = {
   digital_storage_security: [ 'encryption', 'acl' ]
 };
 
-const ASSET_FIXTURE_URL = 'http://localhost:8000/assets/e20f4cd4-9f97-4829-8178-476c7a67eb97/';
+const ASSET_FIXTURE_URL = ENDPOINT_ASSETS + 'e20f4cd4-9f97-4829-8178-476c7a67eb97/';
```

Comment:
I'd be inclined to use "http://iar-backend.invalid/" as the endpoint here. Then any accidental "real" fetches won't risk finding a real server.